currentscape/version.py

```diff
@@ -14,5 +14,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = "0.1.1.dev11"
+VERSION = "0.1.1.dev12"
```

Comment:
Can you please add versioneer, so that we don't have to this manually.

Reply:
I will, good point